docs/guide/tour.rst

```diff
@@ -204,38 +208,44 @@ To get the numerical value of the solar radius in kilometers - without the unit
     con.radius.to('km').value
     695508.0
 
-Quantities and units are simple and powerful tools for keeping track of the units you're working in, and make it
-easy to convert the same physical quantity into different units.  To learn more about the capabilities of quantities
-and units, please consult `the astropy tutorial <http://www.astropy.org/astropy-tutorials/Quantities.html>`__.
-SunPy's approach to the adoption of quantities and units in the codebase is described
-`here <https://github.com/sunpy/sunpy-SEP/blob/master/SEP-0003.md>`__.
+Quantities and units are simple and powerful tools for keeping track of the
+units you're working in, and make it easy to convert the same physical quantity
+into different units. To learn more about the capabilities of quantities and
+units, please consult `the astropy tutorial
+<http://www.astropy.org/astropy-tutorials/Quantities.html>`__.
 
-Here's a simple example of the power of units.  Suppose you have the radius of a circle and would like to calculate
-its area.  The following code implements this ::
+Here's a simple example of the power of units. Suppose you have the radius of a
+circle and would like to calculate its area. The following code implements
+this::
 
     >>> import numpy as np
     >>> import astropy.units as u
     >>> @u.quantity_input(radius=u.m)
     ... def circle_area(radius):
     ...     return np.pi * radius ** 2
 
-The first line imports numpy, and the second line imports astropy's units module.  The beginning of the third line (the
-"@" symbol) indicates that what follows is a Python decorator.  In this case, the decorator allows us to specify what
-kind of unit the function input variable "radius" in the following function "circle_area" should have.  In this case,
-it is meters.  The decorator checks that the input is convertible to the units specified in the decorator.  Calculating
-the area of a circle with radius 4 meters using the function defined above is simple ::
+The first line imports numpy, and the second line imports astropy's units
+module. The beginning of the third line (the "@" symbol) indicates that what
+follows is a Python decorator. In this case, the decorator allows us to specify
```

Comment:
Might be worth introducing units and decorators separately rather than both at once, to reduce cognitive overhead.